src/main/java/org/broadinstitute/hellbender/tools/variantdb/SampleList.java

```diff
@@ -81,8 +81,13 @@ private TableResult querySampleTable(String fqSampleTableName, String whereClaus
                 "SELECT " + SchemaUtils.SAMPLE_ID_FIELD_NAME + ", " + SchemaUtils.SAMPLE_NAME_FIELD_NAME +
                 " FROM `" + fqSampleTableName + "`" + whereClause;
 
+        Map<String, String> labelForQuery = new HashMap<String, String>();
+        labelForQuery.put("gvs_tool_name", "sample-list-creation");
```

Comment:
I would remove this, or somehow take the label as a parameter and pass it in.  There is no sample list creation tool... this is a helper function used by several tools

Reply:
hmmmm I assumed we got charged for the query so it would be a line in the billing table---but is that not true?